e2e/specs/create.workspace.spec.js

```diff
@@ -1,22 +1,30 @@
 const SingIn = require('../pages/SignIn');
-let username = 'hapsneeze';
-let password = 'test12345';
-let testindex = 100;
-describe('pivotal tracker page', () => {
-
+const APIrequest = require('../rest-api/RequestManager');
+const Config = require('../../config.json');
+describe('pivotal tracker page create workspaces', () => {
     let dashboard;
-    let workspace;
-    let workspaceName = 'TestWorkspace'+Math.floor(Math.random() * testindex);
+    let workspaceData = {name: 'Test Workspace '+ new Date().getMilliseconds()};
     //Sign in precondition
     before(() => {
-        dashboard = SingIn.loginAs(username, password);
+        dashboard = SingIn.loginAs(Config.username, Config.password);
     });
 
-    it('should create a new workspace named: '+workspaceName, () => {
+    it('should create a new workspace', () => {
+        let workspace = dashboard.clickCreateWorkspaceButton();
         //create workspace
-        workspace = dashboard.createWorkspace(workspaceName);
-        expect(workspaceName).to.equal(workspace.getWorkspaceName());
+        workspace.createWorkspace(workspaceData);
+        expect(workspaceData.name).to.equal(workspace.getWorkspaceName());
+    });
+
+    after(() => {
+        let response = browser.call(() => {return APIrequest.GetRequest('/my/workspaces');});
+        let workspaceID;
+        Object.values(response.data).map((workspace) => {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this function to use "return" consistently. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3801)